app/views/newfromtemplate.html

```diff
@@ -16,6 +16,7 @@
                   {{ emptyMessage }}
                 </div>
                 <div class="osc-form" ng-show="template">
+                  <alerts alerts="alertsTop" hide-close-button="true"></alerts>
```

Comment:
This feels like a general problem with our alerts we might try and solve. Assuming it can happen with other forms, too.